### ISSUE TITLE
Add default placeholder image when profile image is missing or broken

### ISSUE THREAD

**Issue by abhi-battula** [jenkins-infra/contributor-spotlight#500]
Currently, when a profile image is missing or broken, we rely on the alt text. It would be better to show a default placeholder image instead, to improve the UI and user experience.

We can add a default image that will be displayed whenever the original image fails to load or is not available.

**Comment by abhi-battula** [comment]
Hi kris ,
I’ve created this issue based on the [previous discussion](https://github.com/jenkins-infra/contributor-spotlight/pull/472/changes#r2779503671). I’d be happy to work on it.

**Comment by lemeurherve** [comment]
In which case(s) do you think a missing image could happen? Has it ever occurred?
